neqo-transport/src/packet.rs

```diff
@@ -570,6 +570,7 @@ pub fn encode_packet(crypto: &dyn CryptoCtx, hdr: &PacketHdr, body: &[u8]) -> Ve
 #[allow(unused_variables)]
 mod tests {
     use super::*;
+    use neqo_common::matches;
```

Comment:
How is this change necessary?

Reply:
I updated rust and it started to complain about ambiguity.